source/content/guides/drupal/drupal-hosted/11-troubleshooting.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Migrate a Drupal 8 Site to the Latest Version of Drupal
```

Comment:
@stovak , still contains multiple references to D9.  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted/troubleshooting